tasks/Centos.yml

```
@@ -0,0 +1 @@
+Redhat.yml
```

I'm not 100% sure but believe it should be `RedHat` ?
```suggestion
RedHat.yml
```

---

[Link](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html#ansible-facts-os-family) I believe you are correct 👍